### PR TITLE
release 0.5.3

### PR DESCRIPTION
**Pull request by edgarriba**
### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### PR Implementer
This is a small checklist for the implementation details of this PR.

If there are any questions regarding code style or other conventions check out our 
[summary](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst).

- [ ] Did you discuss the functionality or any breaking changes before ?
- [ ] **Pass all tests**: did you test in local ? `make test`
- [ ] Unittests: did you add tests for your new functionality ?
- [ ] Documentations: did you build documentation ? `make build-docs`
- [ ] Implementation: is your code well commented and follow conventions ? `make lint`
- [ ] Docstrings & Typing: has your code documentation and typing ? `make mypy`
- [ ] Update notebooks & documentation if necessary

### KorniaTeam
<details>
  <summary>KorniaTeam workflow</summary>
  
  - [ ] Assign correct label
  - [ ] Assign PR to a reviewer
  - [ ] Does this PR close an Issue? (add `closes #IssueNumber` at the bottom if 
        not already in description)

</details>

### Reviewer
<details>
  <summary>Reviewer workflow</summary>

  - [ ] Do all tests pass? (Unittests, Typing, Linting, Documentation, Environment)
  - [ ] Does the implementation follow `kornia` design conventions?
  - [ ] Is the documentation complete enough ?
  - [ ] Are the tests covering simple and corner cases ?
 
</details>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,23 @@ All notable changes to this project will be documented in this file.
...
+- Added inverse for augmentations ([#1013](https://github.com/kornia/kornia/pull/1013))
+- Add more advenced augmentations ([#1015](https://github.com/kornia/kornia/pull/1015)
+
+### Fixed
+- Correct Sobel test_noncontiguous. Nothing was tested before. ([#1018](https://github.com/kornia/kornia/pull/1018))
+- Fixing #795: find_homography_dlt_iterated sometimes fails ([#1022](https://github.com/kornia/kornia/pull/1022))
+
+### Changed
+- PoC: build multi-OS Python ([#1017](https://github.com/kornia/kornia/pull/1017)) 
+- Refactorization of the morphology package ([#1034](https://github.com/kornia/kornia/pull/1034))
```
Comment (ducha-aiki) [inline CHANGELOG.md:19]: Add that it is a breaking change, as the function behaviour is changed - before it was not correct

**Review by ducha-aiki — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ducha-aiki) [reply CHANGELOG.md:19]: Morphology, I mean

**Comment by Borda** [comment]
I do not think that the release workflow is worse to mention in changelog =)

File: CHANGELOG.md (hunk 1 above)
Comment (edgarriba) [reply CHANGELOG.md:19]: in theory breaking changes are included in this category. We are following this convention: https://keepachangelog.com/en/1.0.0/
